snmp/datadog_checks/snmp/utils.py

```diff
@@ -74,6 +84,24 @@ def recursively_expand_base_profiles(definition):
         definition.setdefault('metric_tags', []).extend(base_definition.get('metric_tags', []))
 
 
+def get_default_profiles():
+    """Return all the profiles installed on the system."""
+    # type: () -> Dict[str, Any]
+    profiles = {}
+    paths = [_get_profiles_site_root(), _get_profiles_confd_root()]
+    for path in paths:
+        if not os.path.isdir(path):
+            continue
+        for filename in os.listdir(path):
+            if filename.startswith('_'):
```

Comment:
Okay, so this means users won't be able to `extend` from our mixin profiles such as `_generic-if.yaml`, etc, correct? I'm okay with that idea, as we should keep as much control over these as possible. :+1:

Reply:
No they can. This is only for profile matching, which requires a sysobjectid.